recipes/eeglabio/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "eeglabio" %}
+{% set version = "0.0.1.post6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/eeglabio-{{ version }}.tar.gz
+  sha256: fa19c2bb9468f03fdc109c146cd2dabdf224ed9308cf81906635adf428d2e04c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy
+    - python >=3.6
+    - scipy
+
+test:
+  imports:
+    - eeglabio
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jackz314/eeglabio
+  summary: I/O support for EEGLAB files in Python
+  dev_url: https://github.com/jackz314/eeglabio
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jackz314
+    - larsoner
+    - hoechenberger
```

Comment:
@jackz314 and @hoechenberger can you please comment here that you're willing to maintain this conda-forge package?

Reply:
I'm willing to co-maintain as well.